radicle-httpd/src/api/json.rs

```diff
@@ -36,20 +37,21 @@ pub(crate) fn blob(blob: &Blob, path: &str) -> serde_json::Value {
 }
 
 /// Returns JSON for a tree with a given `path` and `stats`.
-pub(crate) fn tree(tree: &Tree, path: &str, stats: &Stats) -> serde_json::Value {
-    let prefix = std::path::Path::new(path);
+pub(crate) fn tree(tree: &Tree, path: &str, stats: &Stats) -> Value {
+    let prefix = Path::new(path);
     let entries = tree
         .entries()
         .iter()
         .map(|entry| {
             json!({
                 "path": prefix.join(entry.name()),
                 "name": entry.name(),
-                "lastCommit": serde_json::Value::Null,
```

Comment:
I think this is supposed to be the whole commit, not just the id!

Reply:
hm... looks like in interface, `lastCommit` is retrieve from `blob` end point and is not being used anywhere else, I think we should remove it.